<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatisSayfasi</class>
 <widget class="QMainWindow" name="SatisSayfasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>588</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>AYLIK TOPLAM SATIŞ TUTARI :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_toplamtutar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kaydet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tarih_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Satış No Ara:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ara">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Güncelle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sil</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_satis">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabloyu Yükle</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>41</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_satisno">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Satış No</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_id"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_ayaricesidi">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Ayarı ve Çeşidi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ayarivecesidi"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_gram">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Gram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_gram"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_satisfiyati">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Satış Fiyatı</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_satisfiyati"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_tarih_3">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Personel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_personel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_tarih">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Tarih</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_tarih"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_ara">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ara</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trash">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resource/img/trash.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionPrinter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrinter">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
